client/nginx.go

```diff
@@ -88,6 +89,18 @@ type Stats struct {
 	StreamUpstreams   StreamUpstreams
 }
 
+// Status contains basic system status information
```

Comment:
Better to use the following doc string: "Status contains general information about NGINX Plus." 
 See also http://nginx.org/en/docs/http/ngx_http_api_module.html#def_nginx_object, although here it refers to NGINX not NGINX Plus